.github/workflows/release.yaml

```diff
@@ -333,3 +333,39 @@ jobs:
       base64-subjects: "${{ needs.combine_hashes.outputs.hashes }}"
       upload-assets: true # Optional: Upload to a new release
       provenance-name: kargo-cli.intoto.jsonl
+
+  update-homebrew-formula:
+    if: github.event_name == 'release' && github.event.action == 'released' # Runs only for stable releases
+    runs-on: ubuntu-latest
+    needs: [ publish-image, publish-cli ]
+    steps:
+    - name: Checkout tap repository
+      uses: actions/checkout@v4
+      with:
+        repository: akuity/homebrew-tap
+        token: ${{ secrets.HOMEBREW_TOKEN }}
+
+    - name: Setup Git User
+      uses: fregante/setup-git-user@v2
+      with:
+        name: ${{ github.actor }}
+        email: ${{ github.actor }}@users.noreply.github.com
```

Comment:
I think this configuration is redundant, based on reading the README of the action.

Reply:
My point was more with this bit from the [action documentation](https://github.com/fregante/setup-git-user?tab=readme-ov-file#usage):

> New commits and tags will be assigned to the [@actions](https://github.com/actions) user. **If you want to customize the user, you don't need this action.**